examples/analytics.amp.html

```diff
@@ -237,6 +237,29 @@
 </amp-analytics>
 <!-- End comScore example -->
 
+<!-- INFOnline example
+
+  Please note: the url variable must point to an HTML file including the ivw
```

Comment:
no longer need as I removed the whole comment - but added the link in the readme.